### ISSUE TITLE
The position of the title

### ISSUE THREAD

**Issue by bt4ibwem8** [stpnwf/ZestyTheme#9]
The name of the title is too high, so it looks no good. Screenshot is from the web browser but it looks similar also in Android app. Server is 10.9.11 version. More polished should be above the year (red arrow):

![jf](https://github.com/user-attachments/assets/10465a85-7de2-47db-88f1-b74e8455b5c9)

**Comment by stpnwf** [comment]
Hmm, seems to happen when there is no logo. Thanks for reporting it. 
I will work on it this weekend when I have time.

**Comment by stpnwf** [comment]
Alright, I pushed the fix to github. Give it a day or so for it to reach github's CDN and make it downstream to you.

**Comment by bt4ibwem8** [comment]
It is better, but still too high:

![image](https://github.com/user-attachments/assets/1e8a59a4-6b54-4929-8cad-9331d2f80301)

**Comment by stpnwf** [comment]
Yeah, but if I move it down too much, there will be too big a gap at the top, since I wanna have a constant placement of the play button, etc. All my media has logos, so this is rly an issue when opening actor pages. Will consider it as well.

Add this in the mean time.

```
.nameContainer .itemName.infoText.parentNameLast > bdi {
	top: 65%;
}
```
